src/guacenc/ffmpeg-compat.h

```diff
@@ -78,5 +78,70 @@
  */
 int guacenc_avcodec_encode_video(guacenc_video* video, AVFrame* frame);
 
+/**
+ * Creates and sets up the AVCodecContext for the appropriate version
+ * of libavformat installed
+ *
+ * @param stream
+ *     The open AVStream
```

Comment:
The format Doxygen-style comments that are used within guacamole-server to document the various functions is similar to JavaDoc and JSDoc in the need for periods, etc. The descriptions of parameters and the description of the function all need to end with a final `.`. For example:

```c
/**
 * Creates and sets up the AVCodecContext for the appropriate version
 * of libavformat installed.
 *
 * @param stream
 *     The open AVStream.
```